tasks/main.yml

```diff
@@ -28,11 +28,14 @@
     - pkgs
     - libs
 
-- name: "Symlink the librato module into the python module directory"
-  file: >
-    src=/opt/collectd-librato/lib/collectd-librato.py
-    dest=/opt/collectd-python-modules/collectd-librato.py
-    state=link
+- name: "Upload Librato collectd config"
+  template: >
+    src=librato.conf.j2
+    dest=/etc/collectd.d/librato.conf
+    owner=root
+    group=root
+    mode=0644
+  when: use_librato
```

Comment:
should also contain `if collectd_forwarder is defined and collectd_forwarder == inventory_hostname`